CMakeLists.txt

```diff
@@ -83,9 +83,9 @@ if(WIN32)
 	enable_language(RC)
 	set(CMAKE_RC_COMPILER windres)
 	file(
-		DOWNLOAD "https://github.com/TheElixZammuto/sunshine-prebuilt/releases/download/1.0.0/pre-compiled.zip" "${CMAKE_CURRENT_BINARY_DIR}/pre-compiled.zip"
+		DOWNLOAD "https://github.com/SunshineStream/Sunshine-Windows-Prebuilt/releases/download/v1/pre-compiled.zip" "${CMAKE_CURRENT_BINARY_DIR}/pre-compiled.zip"
```

Comment:
This should be changed to 

`DOWNLOAD "https://github.com/LizardByte/ffmpeg-prebuilt/releases/download/v1/pre-compiled-windows.zip" "${CMAKE_CURRENT_BINARY_DIR}/pre-compiled.zip"`

I also do not know if that is ffmpeg4 or ffmpeg5, or if it has everything required for Sunshine. I think it does for Windows. That repo needs some help as well to get proper builds of ffmpeg for Windows (and hopefully for Linux and Mac). Finally statically linking those for Linux and Mac would be a huge benefit.